Gemfile.lock

```diff
@@ -277,13 +279,14 @@ GEM
     omniauth-twitter (1.4.0)
       omniauth-oauth (~> 1.1)
       rack
+    parallel (1.12.0)
     parser (2.4.0.0)
       ast (~> 2.2)
-    pg (0.20.0)
+    pg (0.21.0)
```

Comment:
Please don't perform `bundle update` since some dependency versions (e.g. `pg`) are currently broken 😞

Reply:
Okay, sorry about that. Do you want me to make another new PR instead?